Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -16,6 +16,23 @@ final class SettingsViewModel {
     
     var vehicles = [Vehicle]()
     
+    var sortedContributors: [Contributor] {
+        guard let contributors = contributors, !contributors.isEmpty else {
+            return []
+        }
+        
+        return contributors.sorted { (contributor1, contributor2) in
+            switch (contributor1.contributions, contributor2.contributions) {
+            case let (contributor1, contributor2) where contributor1 > contributor2:
+                return true
+            case let (contributor1, contributor2) where contributor1 < contributor2:
+                return false
+            default:
+                return contributor1.login < contributor2.login
+            }
+        }
+    }
+    
```

Comment:
This shouldn't be changed at all, you need to update your fork before making a new branch for new issues